apps/condo/pages/meter/create.tsx

```diff
@@ -0,0 +1,48 @@
+import Head from 'next/head'
+import React from 'react'
+import { useIntl } from '@core/next/intl'
+import { PageContent, PageWrapper } from '@condo/domains/common/components/containers/BaseLayout'
+import { OrganizationRequired } from '@condo/domains/organization/components/OrganizationRequired'
+import { Typography, Row, Col } from 'antd'
+import { ReturnBackHeaderAction } from '@condo/domains/common/components/HeaderActions'
+import { CreateMeterReadingsForm } from '@condo/domains/meter/components/CreateMeterReadingsForm'
+import { useOrganization } from '@core/next/organization'
+
+interface ICreateContactPage extends React.FC {
+    headerAction?: JSX.Element
+    requiredAccess?: React.FC
+}
+
+const CreateMeterPage: ICreateContactPage = () => {
+    const intl = useIntl()
+    const PageTitle = intl.formatMessage({ id: 'meter.AddMeterReadings' })
+
+    const { organization, link: { role: role } } = useOrganization()
+
+    return (
+        <>
+            <Head>
+                <title>{PageTitle}</title>
+            </Head>
+            <PageWrapper>
+                <PageContent>
+                    <Row gutter={[12, 40]}>
+                        <Col span={24}>
+                            <Typography.Title level={1} style={{ margin: 0 }}>{PageTitle}</Typography.Title>
```

Comment:
you sure u need this?
we got global style which should disable this margin

<img width="469" alt="Screenshot 2021-09-14 at 13 54 06" src="https://user-images.githubusercontent.com/18723798/133245163-4765b788-f06f-49ab-bb4c-e02c5304211b.png">